plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/ResultSetViewer.java

```diff
@@ -21,41 +21,110 @@
 import org.eclipse.core.runtime.IStatus;
 import org.eclipse.core.runtime.Status;
 import org.eclipse.core.runtime.jobs.Job;
-import org.eclipse.jface.action.*;
+import org.eclipse.jface.action.Action;
+import org.eclipse.jface.action.ActionContributionItem;
+import org.eclipse.jface.action.GroupMarker;
+import org.eclipse.jface.action.IAction;
+import org.eclipse.jface.action.IContributionItem;
+import org.eclipse.jface.action.IMenuManager;
+import org.eclipse.jface.action.MenuManager;
+import org.eclipse.jface.action.Separator;
+import org.eclipse.jface.action.ToolBarManager;
```

Comment:
Please return this back as it was. I believe these changes are unintended.
We use import with * when there is more than 4 classes of the same package.

Reply:
Generaly, it is not a good idea to export all classes from the package.

Reply:
There is a nice link describing pros and cons of managing includes https://www.baeldung.com/java-wildcard-imports.
I'm voating to define includes obviusly and avoid any conflicting dependency of wildcard using, at same time standard operation of Ctrl+O will work perfectly

Reply:
I'm not deciding for the whole team, just telling you our current rules.